app/services/check_cmc_ohclv_service.rb

```diff
@@ -153,7 +183,7 @@ def create_view!
         password=#{@etl_db_pass}',
         'SELECT coin_key, to_currency, time, volume_to
           FROM staging.#{@table[:name]}
-          WHERE time >= NOW() - #{@table[:interval]}::INTERVAL'
+          WHERE time >= NOW() - ''#{@table[:interval]}''::INTERVAL'
```

Comment:
Hm what's this syntax? This is new to me!

Reply:
The double single quotes escape the single quotes because the interval clause is nested inside a single quoted string.
The ::INTERVAL just casts to interval instead of using `interval ''#{@table[:interval]}''`.
This is just a refactor, where the double single quotes were in each table hash before.